btr-api/src/btr_api/services/bor.py

```diff
@@ -63,40 +63,37 @@ def init_app(self, app: Flask):
     def update_owners(self, submission: Submission, business: dict, token: str) -> requests.Response:
         """Update owners via the bor-api."""
         try:
-            # collect previous parties
-            SubmissionHistory = Submission.__history_mapper__.class_  # pylint: disable=invalid-name
-            old_parties = {}
-            old_submission = db.session.query(SubmissionHistory)\
-                .filter(SubmissionHistory.id == submission.id).order_by(SubmissionHistory.version.desc()).first()
-            if old_submission:
-                for person in old_submission.payload.get('personStatements', []):
-                    old_party_id = person['statementID']
-                    old_parties[old_party_id] = {
-                        'id': old_party_id,
-                        'effectiveDate': submission.payload['effectiveDate']
-                    }
+            # collect previous parties -- TODO: re-evaluate this when doing edit/cease epic
```

Comment:
Usually its good to have ticket, or add info into existing ticket.
Otherwise it might stick longer then we want.